linux_os/guide/services/snmp/snmp_configure_server/snmpd_no_rwusers/rule.yml

```diff
@@ -0,0 +1,30 @@
+documentation_complete: true
+
+prodtype: rhel6,rhel7,rhel8,fedora
+
+title: 'Ensure SNMP Read Write is disabled'
+
+description: |-
+    Edit <tt>/etc/snmp/snmpd.conf</tt>, remove any <tt>rwuser</tt> entries.
+    Once the read write users have been removed, restart the SNMP service:
+    <pre>$ sudo service snmpd restart</pre>
+
+rationale: |-
+    Certain SNMP settings can permit users to execute system behaviors from user
+    writes to the community strings.
+    This may permit a compromised account to execute commands on a remote system.
+
+severity: medium
+
+identifiers:
+    cce@rhel6: 82731-1
+    cce@rhel7: 82732-9
+    cce@rhel8: 82733-7
+    cce@fedora: 82734-5
```

Comment:
It's great that you added the CCEs, but we don't assign CCEs for Fedora.

Reply:
I'll force push with a fix.